lib/rubocop/cop/chef/effortless/depends_chef_vault.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+#
+# Copyright:: 2019, Chef Software Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module RuboCop
+  module Cop
+    module Chef
+      module ChefEffortless
+        # Cookbook:: metadata.rb Chef Vault does not work with Effortless
```

Comment:
```suggestion
        # Chef Vault is not compatible with the Chef Infra Effortless pattern due to its reliance on Data Bags to store secrets.
```

Reply:
Ack! Sorry, that was careless of me. Fixed.